src/test/system_tests/test_ceph_s3.js

```diff
@@ -17,23 +17,28 @@ const promise_utils = require('../../util/promise_utils');
 require('../../util/dotenv').load();
 
 const {
-    mgmt_ip = '127.0.0.1',
+    os = '',
+        pass = '',
+        protocol = 'ws',
+        mgmt_ip = '127.0.0.1',
         mgmt_port = '8080',
         s3_ip = '127.0.0.1',
+        s3_acc_key = 123,
+        s3_sec_key = 'abc',
 } = argv;
 
 
 const api = require('../../api');
 let rpc = api.new_rpc();
 
 let client = rpc.new_client({
-    address: `ws://${mgmt_ip}:${mgmt_port}`
+    address: `${protocol}://${mgmt_ip}:${mgmt_port}` // http/https and give the mgmt ip and port
```

Comment:
isn't the comment should be ws/wss?

Reply:
ws gets socket hang up.. 
wss works fine..